lib/commands/syncinc.js

```diff
@@ -0,0 +1,278 @@
+        ).run()
+      }
+    }
+
+    yield tr.db(targetDB).table(table).indexWait().run()
+  }, 999)
+
+  for (let table of tablesToSync) {
+    let totalRecords = yield sr.db(sourceDB).table(table).count().run()
+    let recordsProcessed = 0
+    let lastRecordsProcessed = 0
+    let perfStat = []
+    let statusInterval = 500
+    let created = 0
+    let updated = 0
+    let deleted = 0
+    let queue = blockingQueue()
+
+    console.log(`Synchronizing ${totalRecords} records in ${table}...                                                                        `)
+    let sourceCursor = yield sr.db(sourceDB).table(table).orderBy({index: sr.asc('id')})
+      .map(function (row) { return {id: row('id'), hash: sr.uuid(row.toJSON())} })
+      .run({cursor: true})
+    let targetCursor = yield tr.db(targetDB).table(table).orderBy({index: tr.asc('id')})
+      .map(function (row) { return {id: row('id'), hash: tr.uuid(row.toJSON())} })
+      .run({cursor: true})
+
+    let si = {}
+    let ti = {}
+
+    si = yield getNextIdx(sourceCursor, si)
+    ti = yield getNextIdx(targetCursor, ti)
+
+    co(function *() {
+      let pc = 0
+      while (pc < 100) {
+        perfStat.unshift(recordsProcessed - lastRecordsProcessed)
+        while (perfStat.length > 30) {
+          perfStat.pop()
+        }
+        let rps = (_.reduce(perfStat, (a, b) => a + b) / (perfStat.length * (statusInterval / 1000))).toFixed(1)
+        pc = ((recordsProcessed / totalRecords) * 100).toFixed(1)
+        process.stdout.write(` RECORDS SYNCHRONIZED: ${recordsProcessed} | ${rps} sec. | %${pc} | created ${created} | updated ${updated} | deleted ${deleted} | concurrency ${queue.concurrency}                    \r`)
+        lastRecordsProcessed = recordsProcessed
+
+        yield Promise.delay(statusInterval)
+      }
+    })
+
+    while (si.id !== Infinity || ti.id !== Infinity) {
+      const cmp = compareValues(si.id, ti.id)
+
+      if (cmp === 0) {  // si.id === ti.id  ->  check hashes
+        let sid = si.id
+        let tid = ti.id
+        if (si.hash !== ti.hash) {
+          yield queue.push(function *() {
+            let record = yield sr.db(sourceDB).table(table).get(sid).run({timeFormat: 'raw'})
+            yield tr.db(targetDB).table(table).get(tid).replace(record).run()
+            updated += 1
+          })
+        }
+        si = yield getNextIdx(sourceCursor, si)
+        ti = yield getNextIdx(targetCursor, ti)
+        recordsProcessed += 1
+      } else if (cmp < 0) {  // si.id < ti.id  ->  copy si
+        let sid = si.id
+        yield queue.push(function *() {
+          let record = yield sr.db(sourceDB).table(table).get(sid).run({timeFormat: 'raw'})
+          yield tr.db(targetDB).table(table).insert(record).run()
+          created += 1
+        })
+        si = yield getNextIdx(sourceCursor, si)
+        recordsProcessed += 1
+      } else if (cmp > 0) {  // si.id > ti.id  ->  delete ti
+        let tid = ti.id
```

Comment:
Is this variable declaration needed?
```suggestion
```

Reply:
Hi!
It seems it is not. Probably I forgot deleting it.
Regards!